src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends AbstractFigure {
+    private double base1;
+    private double base2;
+    private double heigth;
```

Comment:
The variable `heigth` is misspelled. It should be `height` to maintain code clarity and prevent potential bugs.